images/scripts/services.setup

```diff
@@ -32,7 +34,7 @@ docker run -d --rm --name freeipa -ti -h f0.cockpit.lan \
     -e IPA_SERVER_IP=$SERVER_IP \
     -p 53:53/udp -p 53:53 -p 80:80 -p 443:443 -p 389:389 -p 636:636 -p 88:88 -p 464:464 -p 88:88/udp -p 464:464/udp -p 123:123/udp \
     -v /var/lib/ipa-data:/data:Z \
-    freeipa/freeipa-server \
+    freeipa/freeipa-server:fedora-raw-hide \
```

Comment:
The extra dash needs to go. :grin: Please do `./tests-trigger 1238 image:services` after pushing, to trigger a set of representative tests that use that image.

Reply:
> The extra dash needs to go. 

Pilot error! (Ouch)